network-libp2p/src/network.rs

```diff
@@ -915,34 +914,27 @@ impl Network {
                             }
                         }
                     }
-                    NimiqEvent::Ping(event) => {
+                    behaviour::Event::Ping(event) => {
                         match event.result {
                             Err(error) => {
                                 log::debug!(%error, ?event.peer, "Ping failed with peer");
                             }
-                            Ok(PingSuccess::Pong) => {
-                                log::trace!(?event.peer, "Responded Ping from peer");
-                            }
-                            Ok(PingSuccess::Ping { rtt }) => {
-                                log::trace!(
-                                    ?event.peer,
-                                    ?rtt,
-                                    "Sent Ping and received response to/from peer",
-                                );
+                            Ok(duration) => {
+                                log::trace!(?event.peer, ?duration, "Successful ping from peer");
```

Comment:
FYI: `libp2p-ping` already has such a log on `debug` level.

Reply:
Also, version `v0.53` is now using `tracing` instead of `log` so if you are wondering, why you aren't seeing any messages, it is because you need to set up a `tracing-subscriber`.

Reply:
We're also using `tracing`, actually, we just renamed it to `log` so as to not touch every file.

> FYI: libp2p-ping already has such a log on debug level.

Since we cap other crates at the `info!` level usually and sometimes increase the logging level for our crates only, this can still make sense.

Reply:
> We're also using `tracing`, actually, we just renamed it to `log` so as to not touch every file.
> 
> > FYI: libp2p-ping already has such a log on debug level.
> 
> Since we cap other crates at the `info!` level usually and sometimes increase the logging level for our crates only, this can still make sense.

Of course! I just mentioned it so you know you could achieve the same thing with configuration instead of writing the code yourself :)

Each connection also has a span that adds context such as the remote's `PeerId` and address: https://github.com/libp2p/rust-libp2p/blob/0ceb6580941c010edff2a5689ada2e9e33c773cb/swarm/src/connection/pool.rs#L534

Hence, all logs from within the `ConnectionHandler` are automatically scoped to this :)

We've only recently transitioned to this and I'd be very curious to learn how helpful it is to users!